artifacts/attributes.adoc

```diff
@@ -40,6 +40,23 @@
 :rhbk: RHBK
 :keycloak-version: 24.0
 
+// RHTAP information
+:rhtap-version: 1.3
+:rhtap-brand-name: Red Hat Trusted Application Pipeline
+:rhtap-very-short: RHTAP
+
+// TAS information
+:rhtas-brand-name: Trusted Artifact Signer
```

Comment:
I think an additional attribute, such as :rhtas-short: Trusted Artifact Signer, should be included here.
```suggestion
:rhtas-brand-name: Red Hat Trusted Artifact Signer
```